src/components/Cards.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import styles from "../scenes/Game.module.css";
+import { BsArrowLeftSquare, BsArrowRightSquare } from "react-icons/bs";
+import { SharedState } from "src/store/SharedState";
+
+interface CardsProps {
+  sharedState: SharedState;
+}
+
+export const Cards = ({ sharedState }: CardsProps) => {
```

Comment:
This component does not need the entire `SharedState`....can't it just accept a `Spectrum` as a prop?